Mathlib/Data/Setoid/Partition.lean

```diff
@@ -197,6 +197,33 @@ theorem sUnion_classes (r : Setoid α) : ⋃₀ r.classes = Set.univ :=
   Set.eq_univ_of_forall fun x => Set.mem_sUnion.2 ⟨{ y | r.Rel y x }, ⟨x, rfl⟩, Setoid.refl _⟩
 #align setoid.sUnion_classes Setoid.sUnion_classes
 
+/-- The equivalence between the type of equivalence classes of an equivalence relation and its
+quotient. -/
+noncomputable def classesEquivQuotient (r : Setoid α) : Quotient r ≃ Setoid.classes r := by
```

Comment:
```suggestion
noncomputable def quotientEquivClasses (r : Setoid α) : Quotient r ≃ Setoid.classes r := by
```

Reply:
Fixed in the new commit, I also adapted the doc string to reflect the order.